common/probecontrollerinterface.h

```diff
@@ -146,6 +164,27 @@ inline QDataStream &operator>>(QDataStream &in, ObjectId &id)
   return in;
 }
 
+inline QDataStream &operator<<(QDataStream &out, const ObjectIds &ids)
```

Comment:
This looks like the default stream operators for QVector, why do we need an explicit definition of them?